src/chainparams.cpp

```diff
@@ -99,6 +99,7 @@ class CMainParams : public CChainParams {
         consensus.nPosTargetSpacing = 1 * 60; // XSN: 1 minutes
         consensus.nPosTargetTimespan = 60 * 40;
         consensus.nPoSUpdgradeHFHeight = 898488; // 4 December 2019
+        consensus.nTPoSSignatureUpgradeHFHeight = 1264447;
```

Comment:
It's working commenting what's this for, and why.

Reply:
we will set the date later